handlers/main.yml

```diff
@@ -4,16 +4,18 @@
   command: apache2ctl configtest
   notify: [ 'Reload apache' ]
 
-- name: Test apache and restart
-  command: apache2ctl configtest
-  notify: [ 'Restart apache' ]
+# - name: Test apache and restart
+#   command: apache2ctl configtest
+#   notify: [ 'Restart apache' ]
```

Comment:
I understand but I don’t intent to release the role yet so I don’t mind that much. This is just to force me to edit this file and then I should notice the comment that I should also update the part of the docs which refers to this (as documented in this file as well). Thanks for the thoughtful review :+1: